src/window/empire.c

```diff
@@ -128,45 +153,45 @@ static void draw_trade_resource(resource_type resource, int trade_max, int x_off
 static void draw_trade_city_info(const empire_object *object, const empire_city *city)
```

Comment:
Minor layout changes were required to fit buttons vs the old resource icons. However the changes touched a large number of lines. See `Drawbacks #1`